src/Lib/Items/CanvasImage.vala

```diff
@@ -92,6 +92,8 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
                 // the Size component properly calculate the correct size ratio.
                 size.locked = true;
 
+                if (is_height_small ((int)size.width, (int)size.height) || is_width_small ((int)size.width, (int)size.height)) return;
+
```

Comment:
Wrong condition formatting.
Please use the standard formatting:
```
if () {

}
```

Also, write a comment above this condition to explain why we're doing an early return.